test/e2e/serial/tests/workload_placement.go

```diff
@@ -366,8 +366,10 @@ var _ = Describe("[serial][disruptive][scheduler] numaresources workload placeme
 
 			updatedPod = pods[0]
 			By(fmt.Sprintf("checking the pod landed on a node which is different than target node %q vs %q", targetNodeName, updatedPod.Spec.NodeName))
-			Expect(updatedPod.Spec.NodeName).ToNot(Equal(targetNodeName),
-				"pod should not landed on node %q", targetNodeName)
+			if updatedPod.Spec.NodeName == targetNodeName {
+				_ = objects.LogEventsForPod(fxt.K8sClient, updatedPod.Namespace, updatedPod.Name)
```

Comment:
The idea of logging the test pod's logs is nice, but I don't expect something useful to be found here.
It is more useful to log the scheduler's pod logs, so we could see what leads to this decision.

Reply:
here we actually log the pod events, the idea was to check if there is an event for this pod that placed it on the "invalid" node. if scheduler pod logs can help I'll can that.

Reply:
Oh it's the pod events`, I thought it was the pod logs`.
yes, this might be helpful, explaining why the pod land where it landed.